job-server/src/main/resources/application.conf

```diff
@@ -162,3 +162,5 @@ shiro {
     # Time out for job server to wait for authentication requests
     authentication-timeout = 10 s
 }
+
+flyway.locations = "db/h2/migration"
```

Comment:
I would put this under the spark.jobserver hierarchy, and add a comment that this is for migrations.